### PR TITLE
Implements Wood block

### PR DESCRIPTION
**Pull request by Gabriel8579**
Stripped Acacia Wood and Stripped Dark Oak Wood are not working yet

### CODE REVIEW

**Comment by joserobjr** [comment]
Depends on NukkitX#1128 to be fully implemented

File: src/main/java/cn/nukkit/block/BlockWood.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    }
+
+    @Override
+    public int getToolType() {
+        return ItemTool.TYPE_AXE;
+    }
+
+    @Override
+    public boolean canBeActivated() {
+        return true;
```
Comment (HeyZeer0) [inline src/main/java/cn/nukkit/block/BlockWood.java]: Missing a verification to check if the wood is already not stripped!

**Comment by joserobjr** [comment]
As a workaround, adds this to the server plugin folder:
https://github.com/GameModsBR/Workaround-NukkitX-1128/releases/tag/0.0.1

**Comment by Gabriel8579** [comment]
Testing to stripe woods:

- [x] Oak Wood
- [x] Spruce Wood  
- [x] Birch Wood
- [x] Jungle Wood
- [ ] Dark Oak Wood 
- [ ] Acacia Wood

To do: Fix Dark Oak Wood and Acacia Wood.

Stripped woods are dropping normal woods.

File: src/main/java/cn/nukkit/block/BlockWood.java (hunk 1 above)
Comment (Gabriel8579) [reply src/main/java/cn/nukkit/block/BlockWood.java]: Fixed at f27d01599d069e806745d22a28f1759a40e52d7a

**Comment by joserobjr** [comment]
Dependency is resolved, I've merged from NukkiX into here, please remove the temporary plugin from the plugins folder after you update your workspace.

**Comment by joserobjr** [comment]
image

Using this setting in nukkit.yml (uncomment the first generator and comment the second and the options to change to the 1.X format)
```yml
worlds:
 world:
  seed: 404
  #generator: 'FLAT:2;7,17:12,17:13,17:14,17:15,162:12,162:13;1;'
  generator: flat
  options: '2;7,17:12,17:13,17:14,17:15,162:12,162:13;1;'
```

**Comment by joserobjr** [comment]
image

Using this setting in nukkit.yml (uncomment the first generator and comment the second and the options to change to the 1.X format)
```yml
worlds:
 world:
  seed: 404
  #generator: 'FLAT:2;7,17:12,17:13,17:14,17:15,162:12,162:13;1;'
  generator: flat
  options: '2;7,17:12,17:13,17:14,17:15,162:12,162:13;1;'
```

There are more details about the conversion here: Nukkit#1168
Hit me on discord if you need details ^^

**Comment by joserobjr** [comment]
Asking for a new review because a lot of things as changed.

There are more details about the conversion here: NukkitX#1168
Hit me on discord if you need details ^^

**Comment by joserobjr** [comment]
Removed the review tag because I forgot to review this section:
https://github.com/NukkitX/Nukkit/pull/1173/files#diff-9fd86e515349decce8b869b36f93ebe9R69-R81

I'm going to review it tomorrow at night, when it's ready I re-add the review tag.

**Comment by joserobjr** [comment]
Now it's ready for testing again :)

**Pull request merged**